src/shared/ui/Button/Button.tsx

```diff
@@ -26,6 +26,7 @@ export const Button: FC<ButtonProps> = ({
             disabled={disabled}
             onClick={onClick}
             style={style}
+            sx={{ borderColor: '#909090', color: '#909090' }}
```

Comment:
are you sure these styles should be applied for all buttons?
you can create a new prop for Button eg `variant` which will contain different variants for button view: primary, secondary, warning, etc For now we have only primary So when variant is primary you can apply these styles, when a different variant is chosen you can apply different styles